src/main/resources/default/taglib/k/chapter.html.pasta

```diff
@@ -37,8 +37,10 @@
             <i:for type="sirius.biz.tycho.kb.KnowledgeBaseArticle" var="chapter" items="childChapters">
                 <div class="pb-2">
                     <div>
-                        <a href="/kba/@chapter.getLanguage()/@chapter.getArticleId()">
-                            <i class="fa fa-book"></i> @chapter.getTitle()
+                        <a href="/kba/@chapter.getLanguage()/@chapter.getArticleId()"
```

Comment:
Couldn't these use `k:link`? Then it would only be one place where such things have to be fixed in the future